### DISCUSSION TITLE
Is v2 support creating S3 buckets which suitable for S3 Express One Zone?

### DISCUSSION THREAD

**Discussion by Tom910** [aws/aws-cdk#28873] [Q&A]
Hi. Is it possible to use S3 Express One Zone with aws-cdk v2? I haven't found it in the documentation https://docs.aws.amazon.com/AmazonS3/latest/userguide/directory-bucket-create.html . This documentation only have information about cdk v3

**Comment by bestickley** [comment]
Here is sample implementation I created in case in helps anyone: https://github.com/cdklabs/cdk-nextjs/discussions/191#discussioncomment-15537855
